pkg/apis/azuredisk/v1alpha1/zz_generated.deepcopy.go

```diff
@@ -1,4 +1,3 @@
-//go:build !ignore_autogenerated
```

Comment:
Need to restore this line.

Reply:
Looks like `hack/update-codegen.sh` had removed this line. Reverted.